eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -17,35 +17,42 @@ As a front-end developer I would like to:
 * Manage the content via the REST API:
     * Use the REST endpoints to quickly create a new object
     * Use the REST endpoints to quickly update an already created object
-    * Use the REST endpoint
 
 # Creating content POST
 The create content endpoint should be simplified to default most values and the simplest possible.
 
 This call should create:
 * A new content object (not a draft) and publish
 * Assign ownership to the current user
-* Create the object in the default language
+* Create the object in the default language if not set
 * Location can be defined either with ID or URL
 
 ## Example: creating a new object
 
-```
-Resource: /content/object
-Method: POST
+__Resource__: _/content/object_
+
+__Accept__: _application/vnd.ez.api.Content+json_
+
+__Content-Type__: _application/vnd.ez.api.SimpleContentCreate+json_
+
+__Method__: _POST_
+
+__Body__:
+```json
 {
-    "content-location": "/blogs",
-	"content-type": "blog_post",
+    "ParentLocation": <ID> or <URL>,
```

Comment:
Looking to what we have it is currently quite explicit:
```xml
  <LocationCreate>
    <ParentLocation href="/content/locations/1/4/89" />
    <priority>0</priority>
    <hidden>false</hidden>
    <sortField>PATH</sortField>
    <sortOrder>ASC</sortOrder>
  </LocationCreate>
```

What if aim to adjust what we have and allow short forms in the form of _(using xml here for comparison, same applies to json)_
```xml
  <LocationCreate>
    <ParentLocation id="89" />
  </LocationCreate>

<!-- And: --> 
 
  <LocationCreate>
    <ParentLocation urlAliasPath="/blogs" />
  </LocationCreate>
```

Or even simpler like:
```xml
  <LocationCreate parentId="89" />

<!-- And: --> 
 
  <LocationCreate parentUrlAliasPath="/blogs" />
```


However question is maybe for @papelipe and @bdunogier : Would this work also?